frontend/vue/components/Syllabus/SyllabusCreateLayout.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <section class="syllabus-create">
+    <div class="syllabus-create__title-container">
+      <h1 class="syllabus-create__title">
+        {{ $translate("Classroom") }}
+      </h1>
+      <p class="syllabus-create__title-text">
```

Comment:
To me, this class is confusing because the title is text too. Maybe `syllabus-create__title__subtitle` or `syllabus-create__title__description` could be more descriptive. What do you think?

Reply:
good point... will update in my next change